Gems/Atom/Feature/Common/Code/Source/AuxGeom/AuxGeomDrawQueue.cpp

```diff
@@ -565,7 +565,7 @@ namespace AZ
 
         AuxGeomBufferData* AuxGeomDrawQueue::Commit()
         {
-            AZ_ATOM_PROFILE_FUNCTION("AuxGeom", "AuxGeomDrawQueue: Commit");
+            AZ_PROFILE_FUNCTION(AzRender);
```

Comment:
Nit: prefer AZ_PROFILE_SCOPE with the event name (makes for more readable profiles, I'm actually recommending that people avoid AZ_PROFILE_FUNCTION if possible and would like to deprecate it at some point in the future)

I apologize for requesting this since this change occurs in a number of places in this PR but I think it's important for the ergonomics of the profile

Reply:
Completely fair request, the function signatures do look quite ugly in the visualizer